crates/context/src/lib.rs

```diff
@@ -130,6 +130,12 @@ impl Clone for Context {
     }
 }
 
+impl Default for Context {
+    fn default() -> Self {
+        Self::global()
+    }
+}
```

Comment:
Is this change required?

Reply:
I think we should remove this tbh. I dont think Context default should be implemented.